packages/cardano-services/src/Program/programs/types.ts

```diff
@@ -52,14 +53,17 @@ export enum ProviderServerOptionDescriptions {
   UseTypeOrmStakePoolProvider = 'Enables the TypeORM Stake Pool Provider',
   UseBlockfrost = 'Enables Blockfrost cached data DB',
   UseQueue = 'Enables RabbitMQ',
-  PaginationPageSizeLimit = 'Pagination page size limit shared across all providers'
+  PaginationPageSizeLimit = 'Pagination page size limit shared across all providers',
+  HandlePolicyIds = 'Minting Policy ID',
```

Comment:
`HandlePolicyIds` is repeated three times always with the same meaning.

Better to replace any use of `ProjectorOptionDescriptions.HandlePolicyIds` and `ProviderServerOptionDescriptions.HandlePolicyIds` with the (correctly) newly introduced `HandlePolicyIdsOptionDescriptions.HandlePolicyIds`, and remove the first two.

Reply:
af17ac42b57672c085e8d89d7f2565a979aaad0c should fix this